docs/r2/tensorboard_scalars_and_keras_basic.ipynb

```diff
@@ -0,0 +1,452 @@
+        "X_test, Y_test = X[train_size:], Y[train_size:]"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "Je59_8Ts3rq0",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "## Training the model and logging loss\n",
+        "\n",
+        "You're now ready to define, train and evaluate your model. \n",
+        "\n",
+        "To log the *loss* scalar as you train, you'll create the Keras TensorBoard callback, specifying a log directory, and pass it to [Model.fit()](https://https://www.tensorflow.org/api_docs/python/tf/keras/models/Model#fit).\n",
+        "\n",
+        "TensorBoard reads log data from the log directory hierarchy. In this notebook, the root log directory is \"logs/scalars\", suffixed by a timestamped subdirectory. This enables easily identification and selection of training runs as you use TensorBoard and iterate on your model.\n",
+        " "
+      ]
+    },
+    {
+      "metadata": {
+        "id": "VmEQwCon3i7m",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "logdir=\"logs/scalars/\" + datetime.now().strftime(\"%Y%m%d-%H%M%S\")\n",
+        "tensorboard_callback = keras.callbacks.TensorBoard(log_dir=logdir)\n",
+        "\n",
+        "model = keras.models.Sequential([\n",
+        "    keras.layers.Dense(16),\n",
+        "    keras.layers.Dense(1, input_dim=1),\n",
+        "])\n",
+        "\n",
+        "model.compile(\n",
+        "    loss='mse', # keras.losses.mean_squared_error\n",
+        "    optimizer=keras.optimizers.SGD(lr=0.2),\n",
+        ")\n",
+        "\n",
+        "print(\"Training ... With default parameters, this takes less than 10 seconds.\")\n",
+        "training_history = model.fit(\n",
+        "    X_train, # input\n",
+        "    Y_train, # output\n",
+        "    batch_size=train_size,\n",
+        "    verbose=0, # Suppress chatty output; use Tensorboard instead\n",
+        "    epochs=100,\n",
+        "    validation_data=(X_test, Y_test),\n",
+        "    callbacks=[tensorboard_callback],\n",
+        ")\n",
+        "\n",
+        "print(\"Average test loss: \", np.average(training_history.history['loss']))"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "042k7GMERVkx",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "## Examining loss using TensorBoard\n",
+        "\n",
+        "Now, start TensorBoard, specifying the root log directory.\n",
+        "\n",
+        "Wait a few seconds for TensorBoard's UI to spin up. "
+      ]
+    },
+    {
+      "metadata": {
+        "id": "6pck56gKReON",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "%tensorboard --logdir logs/scalars"
```

Comment:
In a text cell under TensorBoard, add an html comment that points to an image of what the static docs should show. Example [here](https://github.com/tensorflow/tensorboard/pull/1870/files#diff-05296e0e7eafc631c60495988e592d11R240).

Reply:
Done.